content/docs/iac/adopting-pulumi/import/_index.md

```diff
@@ -38,7 +38,7 @@ Both approaches allow you to adopt and begin managing existing cloud resources w
 
 Import uses the selected stack's configured [provider](/docs/concepts/resources/providers/) to look up the desired resource in the cloud provider, read its current configuration, and add the resource to the stack to bring it under management by Pulumi from that point forward. For this, it requires two pieces of information:
 
-* The _type_ of cloud resource to import --- either as a type _token_ (a string that uniquely identifies a Pulumi resource type) when using the CLI or as a resource declaration when importing in code. The type token of an Amazon S3 Bucket resource, for example, is `aws:s3/bucketv2:BucketV2`.
+* The _type_ of cloud resource to import --- either as a type _token_ (a string that uniquely identifies a Pulumi resource type) when using the CLI or as a resource declaration when importing in code. The type token of an Amazon S3 Bucket resource, for example, is `aws:s3/bucketV2:BucketV2`.
```

Comment:
Good catch it indeed is this:

https://github.com/pulumi/pulumi-aws/blob/df65048bb98916e5381bfc196653dd7e17db4664/provider/cmd/pulumi-resource-aws/schema.json#L348909